### PR TITLE
fix: replace ltr and rtl with new ms and me

### LINKED ISSUE #8090: [CAL-1411] tailwind refactor: replace ltr and rtl with new ms- and me- prefixes
use this instead:

[https://tailwindcss.com/blog/tailwindcss-v3-3#:\~:text=installation footprint small.-,Simplified RTL support with logical properties,-We’ve made it](https://tailwindcss.com/blog/tailwindcss-v3-3#:\~:text=installation%20footprint%20small.-,Simplified%20RTL%20support%20with%20logical%20properties,-We%E2%80%99ve%20made%20it)

example:

`<div className="ltr:ml-2 rtl:mr-2">`

is now

`<div className="ms-2"  />`

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1411](https://linear.app/calcom/issue/CAL-1411/tailwind-refactor-replace-ltr-and-rtl-with-new-ms-and-me-prefixes)</sub>

### PR DESCRIPTION
**Pull request by joe-shajan**
## What does this PR do?

Fixes #8090

Replaced `ltr` and `rtl` with new `ms` and `me`

`ms` and `me` is not available the tailwind version we are using [I have explained it here](https://github.com/calcom/cal.com/issues/8090#issuecomment-1500272753)

So I have updated tailwind to the latest version which is `3.3.1`

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How should this be tested?

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Review by PeerRich — APPROVED** [review]
code approved, still need to test all paged in ltr and rtl

File: apps/web/package.json [hunk 1]
```diff
@@ -170,7 +170,7 @@
     "module-alias": "^2.2.2",
     "msw": "^0.42.3",
     "postcss": "^8.4.18",
-    "tailwindcss": "^3.2.1",
+    "tailwindcss": "3.3.1",
```
Comment (Udit-takkar) [inline apps/web/package.json:173]: i think we should use "^3.3.1" here.

Comment (PeerRich) [reply apps/web/package.json:173]: agree

Comment (PeerRich) [inline apps/web/package.json:173]:
```suggestion
    "tailwindcss": "^3.3.1",
```

Comment (joeshajan) [reply apps/web/package.json]: > i think we should use "^3.3.1" here.

Thanks, @Udit-takkar I didn't notice it when I update the package.

**Comment by PeerRich** [comment]
awesome work! tested all pages, looks good!

**Comment by joeshajan** [comment]
Thanks Peer 🙂

**Pull request merged**